01/assets/js/app.js

```diff
@@ -1,4 +1,36 @@
 const a = '4.2';
 const b = 9;
 
-console.log(a, b);
\ No newline at end of file
+const number = 20
+
+
+console.log(a, b);
+
+console.log(typeof(a));
+console.log(typeof(b));
+
+let c = parseInt(a)
+console.log(typeof(c));
+
+const getAdd = c + b
```

Comment:
Pamięta, że zmienna to rzeczownik, a nie czasownik dlatego lepiej będzie to nazwa np. `addition`, natomiast `get` sugeruje czynność.